cluster-provision/gocli/opts/node01/node01.go

```diff
@@ -0,0 +1,58 @@
+package node01
+
+import (
+	"embed"
+	"fmt"
+
+	"kubevirt.io/kubevirtci/cluster-provision/gocli/pkg/libssh"
+)
+
+//go:embed conf/*
+var f embed.FS
+
+type Node01Provisioner struct {
+	sshClient libssh.Client
+}
+
+func NewNode01Provisioner(sc libssh.Client) *Node01Provisioner {
+	return &Node01Provisioner{
+		sshClient: sc,
+	}
+}
+
+func (n *Node01Provisioner) Exec() error {
+	cgroupv2, err := f.ReadFile("conf/00-cgroupv2.conf")
+	if err != nil {
+		return err
+	}
+	advAudit, err := f.ReadFile("conf/adv-audit.yaml")
+	if err != nil {
+		return err
+	}
+	cmds := []string{
+		`[ -f /home/vagrant/single_stack ] && export kubeadm_conf="/etc/kubernetes/kubeadm_ipv6.conf" && export cni_manifest="/provision/cni_ipv6.yaml" || { export kubeadm_conf="/etc/kubernetes/kubeadm.conf"; export cni_manifest="/provision/cni.yaml"; }`,
```

Comment:
maybe best to have kept the shell script via embed and used it as a template, and then send it to execute via ssh as a whole for now?

Reply:
Thats fine, but i am trying as much as i can to not create any new bash scripts as this is what we wanted to depart from originally. it might be reasonable if its a long command like that

what do you think ?